isis/src/base/objs/EquatorialCylindricalShape/EquatorialCylindricalShape.h

```diff
@@ -60,11 +65,14 @@ namespace Isis {
       bool intersectSurface(std::vector<double> observerPos,
                             std::vector<double> lookDirection);
 
+    protected:
+      virtual void setTargetRadii(SpiceDouble &a, SpiceDouble &b, SpiceDouble &c);
```

Comment:
We want to avoid passing by reference for this type of method. You should return a list of values or have a member variable that is set inside this method.